src/components/HeaderComponent.tsx

```diff
@@ -8,10 +8,13 @@ export const HeaderComponent = ({ isMobile }: IWidthProps) => {
   const [isMenuOpen, setMenuOpen] = useState<boolean>();
 
   const headerLinks = () => {
+      // todo: fix urls
```

Comment:
https://www.gatsbyjs.com/docs/linking-between-pages/

like this : 

`<Link className="header-component--link"  to="/investors/">Investors</Link>`

But then probably we also need to overwrite the styling again

Reply:
done on borrowers branch